pkg/config/template.go

```diff
@@ -847,6 +852,12 @@ const templateStringCrioRuntimeBlockIOConfigFile = `# Path to the blockio class
 
 `
 
+const templateStringCrioRuntimeBlockIOReload = `# Reload blockio-config-file and rescan blockio devices in the system before applying
+# blockio parameters.
+{{ $.Comment }}blockio_reload = "{{ .BlockIOReload }}"
```

Comment:
I think you need to ditch the quotes here (`blockio_reload` is a boolean).

Reply:
Thanks, @klihub! Fixed.